iceoryx_integrationtest/Readme.md

```diff
@@ -1,44 +1,52 @@
-# Iceoryx Integrationtests
```

Comment:
@dkroenke @FerdinandSpitzschnueffler @MatthiasKillat It might be a good idea to include this on [iceoryx.io](https://iceoryx.io.). What do you think? Integration tests are for me ASPICE SWE.6 corresponding to a feature of iceoryx. We could extend the document and mention this.

Reply:
They are mainly (all?) derived from examples and these are on iceoryx.io I think. From this point of view the integrationtests could be added as well.

But it is no must-have in my book.

Reply:
@mossmaurice @MatthiasKillat Not part of this PR, since this is the validation of the readme. But I added it to #743.